contrib/ci/dependencies.xml

```diff
@@ -5,6 +5,19 @@
       <package />
     </distro>
   </dependency>
+  <dependency type="build" id="secureboot-db">
```

Comment:
I think these should be runtime dependencies too, no?

Reply:
Heh, I forgot all about the `type`. I don't think we support declaring runtime deps in the dependencies.xml manifest do we?

Reply:
I added some bits in the spec file and in `control.in` -- yell if this was wrong.

Reply:
You got it right